yarn.lock

```diff
@@ -2503,6 +2503,7 @@
 "@patternfly/patternfly@2.33.8":
   version "2.33.8"
   resolved "https://registry.yarnpkg.com/@patternfly/patternfly/-/patternfly-2.33.8.tgz#2885847b4c49d8606abed17a0415da24fec222aa"
+  integrity sha512-tkT1GrdIyvGiDvNHATOa7XLMLKdztpra8ngS8qhCcEWBKh9MWNj5ysjOrkna32bSF9GEkR4TFX492W8ESTHWvQ==
```

Comment:
Can you revert the changes in this file.  It should not have been updated with the changes you made.